src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+public enum Fruit {
```

Comment:
don't need to create an enum for Fruit, you can have more than two fruits potentially